go.mod

```diff
@@ -21,7 +21,6 @@ require (
 	github.com/openshift/machine-api-operator v0.2.1-0.20200226185612-9b0170a1ba07
 	github.com/openshift/machine-config-operator v0.0.1-0.20200913004441-7eba765c69c9
 	github.com/openshift/operator-custom-metrics v0.4.2
-	github.com/operator-framework/api v0.3.8
```

Comment:
Not questioning the validity of these changes, but should they be going alongside a doco update PR?

Reply:
hmm, good question, not sure what happened in my local env, maybe I can exclude those unrelated files